src/components/dropdown/dropdown.styles.js

```diff
@@ -1,6 +1,6 @@
 import { makeStyles } from '@material-ui/core';
 
-const useStyles = makeStyles((theme) => ({
+const dropdownStyles = makeStyles(() => ({
```

Comment:
Why we need this renaming?

Reply:
**sonarcloud** has a rule `Default export names and file names should match` and exceptions for it - `_` and `-`. So, I decided to try that naming for one file and see - will it shown as code smell since in file name includes `.`. And if not - maybe we should naming ours components styles hooks as `dropdownStyles` for example.